### DISCUSSION TITLE
Does it support 2D ?

### DISCUSSION THREAD

**Discussion by finscn** [jrouwe/JoltPhysics.js#15] [Q&A]
Can I use it in my 2D game ? 
Is there a switch to disable compute the data in z-axis ?

**Comment by jrouwe** [comment]
Sorry, thought this was a C++ question.

Yes you can do 2D physics:

https://jrouwe.nl/jolt/examples/2d_funnel.html

Had to expose a flag for it to work.

**Reply by finscn** [reply]
the online demo works correct , but when I download the demo and changed the  jolt-physics.wasm to github version , it's wrong.

<img width="514" alt="image" src="https://github.com/jrouwe/JoltPhysics.js/assets/288367/b74bb9b4-04e6-474d-b59e-d263425962b6">

**Reply by jrouwe** [reply]
There is no jolt-physics.wasm file on github, you need to download it from https://www.unpkg.com/ as specified on the main page or perhaps you built it yourself and forgot to run build.sh again?

**Reply by finscn** [reply]
I've build it by myself in macOS 13.5.2 , mbp with M2 pro

**Reply by jrouwe** [reply]
When you say 'when I download the demo' do you mean that you did a pull on the repository (I made source code changes)?
If yes, then try deleting the Build folder and running build.sh again, perhaps something went wrong in dependency checking?
